src/k8s-engine/lima.ts

```diff
@@ -453,6 +455,27 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     }
   }
 
+  protected updateConfigPortForwards(config: LimaConfiguration) {
+    let allPortForwards: Array<Record<string, any>> | undefined = config.portForwards;
+
+    if (!allPortForwards) {
+      // This shouldn't happen, but fix it anyway
+      config.portForwards = allPortForwards = DEFAULT_CONFIG.portForwards ?? [];
+    }
+    const dockerPortForwards = allPortForwards?.find(pair => Object.keys(pair).length === 2 &&
+      pair.guestSocket === '/run/docker.sock' &&
+      ('hostSocket' in pair));
+
+    if (!dockerPortForwards) {
+      config.portForwards?.push({
+        guestSocket: '/run/docker.sock',
+        hostSocket: path.join(paths.lima, MACHINE_NAME, 'docker.sock')
```

Comment:
Please make sure that any symlinks in `path.lima` have been evaluated. If the host username is too long, the lima directory may need to be replaced with a symlink to something with a shorter path.

Reply:
Why is it an issue if `paths.lima` is a symlink? This is on the host side, what's the problem?

Reply:
Explained on slack, but in short: the socket path must be less than 104 characters on macOS (108 on Linux)